src/components/common/Text/Text.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import './Text.css'
+
```

Comment:
Should there be a function that selects what type it is? Like in Flash.js line 5